src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let vocationCost = 0;
+
+  if (days >= 7) {
+    vocationCost += days * cost - bigDiscount;
```

Comment:
Use constants for the number of days that define the thresholds for discounts to avoid hardcoding. For example, 'const LONG_TERM_RENTAL = 7;' and 'const SHORT_TERM_RENTAL = 3;'.